.circleci/config.yml

```diff
@@ -21,14 +21,29 @@ jobs:
       - run:
           name: Install Dependencies
           command: yarn
+      - run:
+          name: Build all code to JavaScript
+          command: yarn build
+      - run: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> ~/.npmrc
```

Comment:
This line is only required when publishing on `npm`, which we're not doing here.